app/models/concerns/s3_headers.rb

```diff
@@ -0,0 +1,22 @@
+module S3Headers
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def s3_headers
+      {
+        s3_headers: {
+          'x-amz-meta-Cache-Control' => 'no-cache',
+          'Expires' => 3.months.from_now.httpdate
+        },
+        s3_permissions: :private,
+        s3_region: region
+      }
+    end
+
+    private
+
+    def region
+      Settings.aws.region || 'eu-west-1'
```

Comment:
this will work fine on the kubernetes branch but merging the resulting kubernetes branch to master (which i think we are looking to do) would break current s3 by the looks of it.

I would think that setting the Settings.aws.region in settings to ENV['AWS_REGION'] then removing the || clause here would do it?!

TD envs have a `AWS_REGION` and looks like k8s secrets do  too (though why it is a secret is another matter and now has two REGION keys)
```
$ k exec -it claim-for-crown-court-defence-576d8b6cb-dqnwr sh
/usr/src/app $ env | grep REGION
AWS_REGION=eu-west-2
SETTINGS__AWS__S3__REGION=eu-west-2
```

Reply:
On TD at the moment...
```$ sudo docker exec -it advocatedefencepayments sh
/usr/src/app $ env | grep REGION
AWS_REGION=eu-west-1```